### DISCUSSION TITLE
Model names do not accept '+' symbol

### DISCUSSION THREAD

**Discussion by neosunrise** [EdgeTX/edgetx#1433] [Q&A]
Today I was trying to add "Champ S+" to the radio but it seems that I cannot enter the plus sign in the name, nor can I copy and paste in Companion. Is it a known issue? I can type it without issues on my Spektrum radios.

**Comment by pfeerick** [comment]
It looks like `+` is not a valid character atm, which is also why Companion will not let you enter or copy it in (since it would be silly for Companion to let you enter invalid characters). Will have to see why that is.

**Reply by elecpower** [reply]
Could be a carry over limitation from zchar though I can envisage it playing havoc with char manipulation in one of the coding languages used.
